client/src/app/nav-menu/nav-menu.component.html

```diff
@@ -21,10 +21,14 @@
       >
         <ul class="navbar-nav flex-grow">
           <li class="nav-item" [routerLinkActive]="['link-active']">
-            <a class="nav-link text-dark" [routerLink]="['/fridge']">Fridge</a>
+            <a
+              class="nav-link text-dark"
+              [routerLink]="['/{{ appPaths.fridge }}']"
```

Comment:
Small bug

Variables for routerLinks are added by doing `[routerLink]="['/' ,appPaths.fridge]"` , received an error while trying to access fridge on nav-menu

Reply:
Thanks for pointing it out, fixed it.